staking/availability/measure.go

```diff
@@ -234,29 +249,29 @@ func compute(
 		return err
 	}
 
-	signed, toSign, quotient, err := ComputeCurrentSigning(snapshot, wrapper)
+	computed, err := ComputeCurrentSigning(snapshot, wrapper, blocksPerEpoch)
 
 	if err != nil {
 		return err
 	}
 
 	utils.Logger().Info().
-		Str("signed", signed.String()).
-		Str("to-sign", toSign.String()).
-		Str("percentage-signed", quotient.String()).
-		Bool("meets-threshold", quotient.LTE(measure)).
+		Str("signed", computed.Signed.String()).
+		Str("to-sign", computed.ToSign.String()).
+		Str("percentage-signed", computed.Percentage.String()).
+		Bool("meets-threshold", computed.IsBelowThreshold).
 		Msg("check if signing percent is meeting required threshold")
 
 	const missedTooManyBlocks = true
 
-	switch IsBelowSigningThreshold(quotient) {
+	switch computed.IsBelowThreshold {
 	case missedTooManyBlocks:
-		wrapper.EPOSStatus = effective.Inactive
+		wrapper.Status = effective.Inactive
 		utils.Logger().Info().
 			Str("threshold", measure.String()).
 			Msg("validator failed availability threshold, set to inactive")
 	default:
-		wrapper.EPOSStatus = effective.Active
+		wrapper.Status = effective.Active
```

Comment:
please add a TODO： we need to take care of the situation when a validator wants to stop validating, but if he turns his validator to inactive and his node is still running, the status will be turned back to active automatically.  Don't need to address in this PR.